assets/js/modules/analytics/datastore/properties.js

```diff
@@ -151,6 +151,8 @@ const baseActions = {
 				return;
 			}
 
+			// Clear any profile ID selection in the case that selection falls to the getProfiles resolver.
+			registry.dispatch( STORE_NAME ).setProfileID( '' );
 			registry.dispatch( STORE_NAME ).setPropertyID( propertyID );
 
 			if ( PROPERTY_CREATE === propertyID ) {
```

Comment:
That's unrelated, but shouldn't there also be a call to `setInternalWebPropertyID`, making it empty, for this case? Otherwise, when selecting to create a new property, any previous internal web property ID will still be in state, potentially causing problems.

Reply:
Yes, it was a bit confusing. I have updated it to be more properly organized. Does it look good to you?

Reply:
This looks better, setting it all at the same time. However, my one point above about the `internalWebProperty` not being set here could still cause problems IMO. But that's not actually in the scope of this issue, so it's okay.